_bookdown.yml

```
@@ -1,42 +1,43 @@
-book_filename: "DocumentationR"
-output_dir: "./_pagedown_output/"
-clean: true
-delete_merged_file: true
-rmd_files: 
-  - index.Rmd
-  - 01_R_Insee/Fiche_utiliser_utilitR.Rmd
-  - 01_R_Insee/01-intro.Rmd
-#  - 01_R_Insee/Fiche_utiliser_Rstudio_SSPCloud.Rmd
-#  - 01_R_Insee/Fiche_installer_packages.Rmd
-  - 01_R_Insee/Fiche_utiliser_ressources.Rmd
-  - 01_R_Insee/Travail_avec_R_Insee.Rmd
-  - 03_Fiches_thematiques/3-Intro_partie_fiche_thematiques.Rmd
-  - 03_Fiches_thematiques/Modele_de_fiche.Rmd
-  - 03_Fiches_thematiques/Fiche_import_fichiers_plats.Rmd
-  - 03_Fiches_thematiques/Fiche_import_tables_sas.Rmd
-  - 03_Fiches_thematiques/Fiche_import_tableurs.Rmd
-  - 03_Fiches_thematiques/Fiche_joindre_donnees.Rmd
-  - 03_Fiches_thematiques/Fiche_connexion_bdd.Rmd
-  - 03_Fiches_thematiques/Fiche_tidyverse.Rmd
-  - 03_Fiches_thematiques/Fiche_datatable.Rmd
-  - 03_Fiches_thematiques/Fiche_survey.Rmd
-  - 03_Fiches_thematiques/Fiche_rmarkdown.Rmd
-  - 03_Fiches_thematiques/Fiche_graphiques.Rmd
-  - 03_Fiches_thematiques/Fiche_donnees_textuelles.Rmd
-  - 03_Fiches_thematiques/Fiche_analyse_de_donnees.Rmd
-  - 03_Fiches_thematiques/Fiche_donnees_spatiales.Rmd
-  - 03_Fiches_thematiques/Fiche_comment_choisir_un_package.Rmd
-#  - 03_Fiches_thematiques/3-fiches_thematiques.Rmd
-#  - 02_Bonnes_pratiques/2-Intro_partie_bonnes_pratiques.Rmd
-#  - 02_Bonnes_pratiques/02-nommage.Rmd
-#  - 02_Bonnes_pratiques/03-syntaxe.Rmd
-#  - 02_Bonnes_pratiques/04-organisation.Rmd
-#  - 02_Bonnes_pratiques/05-regles.Rmd
-#  - 02_Bonnes_pratiques/06-outils.Rmd
-language:
-  ui:
-    chapter_name: "Chapitre "
-before_chapter_script: [".Rprofile"]
-view: https://github.com/InseeFrLab/utilitR/blame/master/%s
-edit: https://github.com/InseeFrLab/utilitR/edit/master/%s
-history: https://github.com/InseeFrLab/utilitR/commits/master/%s
+book_filename: "DocumentationR"
+output_dir: "./_pagedown_output/"
+clean: true
+delete_merged_file: true
+rmd_files: 
+  - index.Rmd
+  - 01_R_Insee/Fiche_utiliser_utilitR.Rmd
+  - 01_R_Insee/01-intro.Rmd
+  - 01_R_Insee/Travail_avec_R_Insee.Rmd

```

Je me suis trompé en réglant le conflit dans ce fichier, c'est cette fiche la bonne
```suggestion
  - 01_R_Insee/Fiche-personnaliser-R.Rmd
```